pkg/order/controller.go

```diff
@@ -121,6 +125,13 @@ func (c *Controller) calculateProfit(o *model.Order) (value, percent float64, er
 		} else {
 			quantity = math.Max(quantity-order.Quantity, 0)
 		}
+
+		// Fees are often determined by 30day volume.
+		// binance starts 0.001
+		// others at 0.0025
+		// We keep track of volume to have an indication of costs. (0.001%) binance.
+		// TODO do someting with trading fee
```

Comment:
This fee is retrieved when the bot starts. We already have it.

Reply:
But is it calculated with summary? Does the paper wallet do something with the fee? I could not find it yet.  So i wanted to keep track of volume first and then maybe be able to add cost  on the amount of volume you moved in the last 30 days like the real brokers.

Maybe add add the fee to each order and keep track of that is a better solution. The paper wallet has nothing now.

Reply:
removed / improved  comment

Reply:
It is retrieved if you have a Binance account configured, in the paper wallet, it is not significant.